chapters/managementCollection.md

```diff
@@ -0,0 +1,16 @@
+# Collection management
+## Dashboard display
+The dashboard is the main page from which you can have access to the list of collection in the system and other features. To enter you must be authenticated.
+![](../img/dashboard.png)
+
+## Opening collection index
+The collection index page displays the contents of a collection in a table. To open a collection index (see Figure 3) you need to click on the link present in the menu bar (1) , MaaS will show the entire contents of the collection (2). This table displays all the documents you can open by clicking on the related links (3).
+![](../img/collectionIndexPage.png)
+
+## Opening the show-page of a document
+The show-page of a document displays the document's attributes in a table. For access a show-page you need to position yourself in the collection index of belonging and follow step 3.
```

Comment:
typo: è "For accessing" oppure "to access".

se il senso era "devi trovarti nell indice della collection di appartenenza" penso che la traduzione giusta sia "you must be in the index of the collection the show page belongs"